src/wp-includes/class-wp-fatal-error-handler.php

```diff
@@ -190,7 +190,11 @@ protected function display_default_error_template( $error, $handled ) {
 			if ( is_multisite() ) {
 				$message = __( 'There has been a critical error on this website. Please reach out to your site administrator, and inform them of this error for further assistance.' );
 			} else {
-				$message = __( 'There has been a critical error on this website. Please check your site admin email inbox for instructions.' );
+				$message = sprintf(
+					/* translators: %s: Support forums URL. */
+					__( 'There has been a critical error on this website. Please check your site admin email inbox for instructions. If you need further assistance, please try the <a href="%s">support forums</a>.' ),
```

Comment:
```suggestion
					__( 'There has been a critical error on this website. Please check your site admin email inbox for instructions. If you continue to have problems, please try the <a href="%s">support forums</a>.' ),
```
For consistency

Reply:
Yes, [that text](https://translate.wordpress.org/projects/wp/dev/en-ca/default/?filters%5Bterm%5D=please+try+the) should be better. Thanks!